.travis.yml

```diff
@@ -40,20 +34,20 @@ before_install:
   - pip install -U pip setuptools wheel
   - pip install --no-cache-dir -U -r requirements-test.txt
   - ./runcheckmigration
-  - flake8
-  - if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; then ./runisort; fi
+  - if [[ $TRAVIS_PYTHON_VERSION != 3.8 ]]; then flake8; fi
```

Comment:
If you wonder why this is here, It is because pyflakes does not currently support python 3.8 so this is always going to fail for python 3.8,

Reply:
let's remove python 3.8 to keep things simple, I want to avoid adding this kind of conditional in all the modules